.github/workflows/test_gpt2_model.yaml

```diff
@@ -0,0 +1,40 @@
+name: Test GPT-2 with pinned versions.
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 21 * * *'
+
+jobs:
+  install_test:
+    name: Install latest iree-jax release and tests GPT-2
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.9'
+
+      - name: Install Python Dependencies
+        run: |
+          python3 -m pip install absl-py future h5py numpy transformers
+
+      - name: Run installation
+        run: |
+          python $GITHUB_WORKSPACE/build_tools/configure.py --use-pinned
+          python3 -m pip install . -f https://github.com/google/iree/releases --force-reinstall .[xla,cpu,test]",
+
+      - name: Fetch GPT-2
+        run: |
+          models/gpt2/setup.sh
+
+      - name: Export GPT-2
+        run: |
+          python $GITHUB_WORKSPACE/models/gpt2/export.py
+
+      - name: Test GPT-2
+        run: |
+          python $GITHUB_WORKSPACE/models/gpt2/test_export.py
```

Comment:
How long does this take on average? Just a few seconds/minutes? If larger, we'd want to be careful about using up too much time on the actions runners, since they are a shared resource throughout the org.

Looks like https://github.com/rsuderman/iree-jax/runs/6264872065?check_suite_focus=true is a sample run? (would be nice to include that in the PR description)

Reply:
About two minutes, it should be short enough that a nightly should be fine. It may be possible to run it only in cases where a version update occurs which is the main goal.